airflow-core/src/airflow/migrations/versions/0069_3_0_0_add_dag_version_to_dag_run.py

```diff
@@ -0,0 +1,52 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Add dag version to dag run.
+
+Revision ID: eea3f3f779e3
+Revises: 29ce7909c52b
+Create Date: 2025-04-11 15:22:20.093294
+
+"""
+
+from __future__ import annotations
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy_utils import UUIDType
+
+revision = "eea3f3f779e3"
+down_revision = "29ce7909c52b"
+branch_labels = None
+depends_on = None
+airflow_version = "3.0.0"
+
+
+def upgrade():
+    """Apply Add dag version to dag run."""
+    with op.batch_alter_table("dag_run", schema=None) as batch_op:
+        batch_op.add_column(sa.Column("created_dag_version_id", UUIDType(binary=False), nullable=True))
+        batch_op.create_foreign_key("dag_version_id_fkey", "dag_version", ["created_dag_version_id"], ["id"])
```

Comment:
Can we update the migration that added dag_version instead of a new migration file?

Reply:
sure